src/mission_companion.cpp

```diff
@@ -185,28 +185,28 @@ bool talk_function::outpost_missions(npc *p, std::string id, std::string title)
     std::string entry, entry_aux;
 
     if (id == "SCAVENGER"){
-        col_missions["Assign Scavenging Patrol"] = "Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
+        col_missions["Assign Scavenging Patrol"] = _("Profit: $25-$500\nDanger: Low\nTime: 10 hour missions\n \n"
```

Comment:
This works with gettext:
```C++
    auto x =_("abd"
            "def\n"
            "gggg"
            "more");
```

For future reference: if you have `gettext` installed, you can try it out. Change the source as you like, run `./lang/update_pot.sh` and look at "lang/po/cataclysm-dda.pot" (should have just changed) and search for the string you added to the source (try it with a hopefully unique string).